### PR TITLE
Add workflow that detects and triggers CC

### PR DESCRIPTION
**Pull request by Brend-Smits**
Signed-off-by: Brend Smits <brend.smits@philips.com>

### CODE REVIEW

File: .github/workflows/trigger-cc-action.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  push:
+    paths:
+      - /data/wip/*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - id: files
+      uses: jitterbit/get-changed-files@v1
```
Comment (JeroenKnoops) [inline .github/workflows/trigger-cc-action.yaml]: What is this doing? If nothing changed exit?

**Review by JeroenKnoops — COMMENTED** [review]
One question..

**Comment by JeroenKnoops** [comment]
You also have to empty the `wip` folder, otherwise it will check it every time..

File: .github/workflows/trigger-cc-action.yaml (hunk 1 above)
Comment (Brend-Smits) [reply .github/workflows/trigger-cc-action.yaml]: Thanks!!

**Review by JeroenKnoops — APPROVED** [review]

**Pull request merged**